charts/graph-network-indexer/templates/indexer-service/deployment.yaml

```diff
@@ -1,7 +1,9 @@
-{{- $values := $.Values.indexerService }}
+{{- $values := merge (deepCopy .Values.indexerDefaults) .Values.indexerService }}
```

Comment:
most of the other merges, mergeOverwrite has been used. Many of the new added ones have used "merge" which only adds new keys that are present in .indexerService but not present in .indexerDefaults: i.e., it never overwrites, which is not what we want.